src/components/VideoThumbnail/VideoThumbnail.scss

```
@@ -67,4 +67,31 @@ $start-button-size: 60px;
   background-color: var(--p-surface, color('ink'));
   opacity: 0.8;
   text-align: center;
+
+  &.WithProgress {
+    margin-bottom: spacing(base-tight);
+  }
+}
```

I'm not 100% sure on this one but I believe this is what this should be like:

```suggestion
}

.withProgress {
  margin-bottom: spacing(base-tight);
}
```